apps/nextjs-app-router/src/makeswift/client.ts

```diff
@@ -7,3 +8,12 @@ export const client = new Makeswift(MAKESWIFT_SITE_API_KEY, {
   runtime,
   apiOrigin: process.env.MAKESWIFT_API_ORIGIN,
 })
+
+export const getComponentSnapshot = async (snapshotId: string) => {
+  // const locale = await getLocale();
+
+  return await client.getComponentSnapshot(snapshotId, {
+    // locale: locale === defaultLocale ? undefined : locale,
```

Comment:
Are the comments intentional?

Reply:
Yeah, it's a TODO, we need to pass a locale here for this to work w/ localization.